pontoon/base/templates/project_selector.html

```diff
@@ -13,33 +13,38 @@
     </div>
     {% if not project %}
     <div class="sort clearfix">
-      <span class="asc">Name<i class="fa"></i></span>
-      <span>Website<i class="fa"></i></span>
-      <span>Progress<i class="fa"></i></span>
+      <span class="name asc">Name<i class="fa"></i></span>
+      <span class="latest">Latest activity<i class="fa"></i></span>
+      <span class="progress">Progress<i class="fa"></i></span>
     </div>
     {% endif %}
     <ul>
       {% for p in projects %}
       <li class="clearfix{% if project and project == p %} current{% endif %}">
-        {% if url_prefix %}
-        <a href="/{{ url_prefix }}/{{ p.slug }}/" class="clearfix">
+        <span class="name"
+          data-slug="{{ p.slug }}"
+          {% if p.details %}data-details="{{ p.details }}"{% endif %}>
+          {% if url_prefix %}<a href="/{{ url_prefix }}/{{ p.slug }}/" class="clearfix">{% endif %}
+            {{ p.name }}
+          {% if url_prefix %}</a>{% endif %}
+        </span>
+        {% if not project %}
+        <span class="latest">
+          {% set latest_activity = p.latest_activity(locale) if locale else p.latest_activity() %}
```

Comment:
Yeah, if the locale is an optional argument anyway, could you just always pass it in? That'd remove the if statement from the template and avoid the issue as well.